.github/workflows/builds.yml

```diff
@@ -2530,7 +2530,7 @@ jobs:
     needs: [ prepare-deps, prepare-cbindgen ]
     strategy:
       matrix:
-        dpdk_version: [ 22.11.4, 21.11.6, 20.11.10, 19.11.14 ]
+        dpdk_version: [ 23.11.2, 22.11.4, 21.11.6, 20.11.10, 19.11.14 ]
```

Comment:
git message should reference https://redmine.openinfosecfoundation.org/issues/6382 (please check other commits for missing ticket references too)